tests/functional/newsletter/test_newsletter_embed.py

```diff
@@ -48,7 +48,7 @@ def test_newsletter_sign_up_fails_when_missing_required_fields(page_class, base_
 
 @pytest.mark.nondestructive
 @pytest.mark.parametrize(('page_class', 'url_kwargs'), [
-    (ContributePage, None),
+    pytest.param(ContributePage, None, marks=pytest.mark.smoke()),
```

Comment:
I don't think this `smoke()` marker is required any longer?

Reply:
Good catch, removed it in f0a3501